From 66a9a36c3bcd5709c30ac1f2be998eea034a9f6d Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Tue, 1 Nov 2022 17:53:42 +0100
Subject: [PATCH] test: rename TEST-26-SETENV to TEST-26-SYSTEMCTL

(cherry picked from commit c5c258ae0a4a0cfc829ed07ff96c7fab79b6ca71)

Related #2138081
---
 test/{TEST-26-SETENV => TEST-26-SYSTEMCTL}/Makefile | 0
 test/{TEST-26-SETENV => TEST-26-SYSTEMCTL}/test.sh  | 2 +-
 test/units/testsuite-26.service                     | 2 +-
 3 files changed, 2 insertions(+), 2 deletions(-)
 rename test/{TEST-26-SETENV => TEST-26-SYSTEMCTL}/Makefile (100%)
 rename test/{TEST-26-SETENV => TEST-26-SYSTEMCTL}/test.sh (79%)

diff --git a/test/TEST-26-SETENV/Makefile b/test/TEST-26-SYSTEMCTL/Makefile
similarity index 100%
rename from test/TEST-26-SETENV/Makefile
rename to test/TEST-26-SYSTEMCTL/Makefile
diff --git a/test/TEST-26-SETENV/test.sh b/test/TEST-26-SYSTEMCTL/test.sh
similarity index 79%
rename from test/TEST-26-SETENV/test.sh
rename to test/TEST-26-SYSTEMCTL/test.sh
index b38e37bfce..64accf850f 100755
--- a/test/TEST-26-SETENV/test.sh
+++ b/test/TEST-26-SYSTEMCTL/test.sh
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 set -e
 
-TEST_DESCRIPTION="test setenv"
+TEST_DESCRIPTION="systemctl-related tests"
 
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
diff --git a/test/units/testsuite-26.service b/test/units/testsuite-26.service
index aa553b61a6..d8fdaffb06 100644
--- a/test/units/testsuite-26.service
+++ b/test/units/testsuite-26.service
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 [Unit]
-Description=TEST-26-SETENV
+Description=TEST-26-SYSTEMCTL
 
 [Service]
 ExecStartPre=rm -f /failed /testok
